build_routereflector/README.md

```diff
@@ -124,3 +122,169 @@ curl --cacert <path_to_ca_cert> --cert <path_to_cert> --key <path_to_key> -L htt
 curl --cacert <path_to_ca_cert> --cert <path_to_cert> --key <path_to_key> -L https://<ETCD_IP:PORT>:2379/v2/keys/calico/bgp/v1/rr_v6/<IPv6_RR> -XPUT -d value="{\"ip\":\"<IPv6_RR>\",\"cluster_id\":\"<CLUSTER_ID>\"}"
 ```
 
+## Using the Route Reflector image (with Kubernetes API datastore)
```

Comment:
Yes it does belong there too -  we have it in the docs.  Here for completeness since we had this doc too.

Docs.projectcalico.org is up next.